### PR TITLE
[Tachyon-1439]Add and implement getUsage method in TfsShellCommand interface

### PR DESCRIPTION
**Pull request by leohjwang**
getUsage method has been added, and TfsShell now uses getUsage in printUsage

### CODE REVIEW

File: shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java [hunk 1]
```diff
@@ -91,4 +91,9 @@ public void run(String... args) throws IOException {
     }
     System.out.println("Lineage " + lineageId + " has been created.");
   }
+
+  @Override
+  public String getUsage() {
+    return "createLineage <inputFiles,...> <outputFiles,...> <cmd>";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java]: I think this should be `<inputFile1,...> <outputFile1,...> [<cmd_arg1> <cmd_arg2> ...]`

**Comment by aaudiber** [comment]
Thanks for the improvement @leohjwang, especially for adding the missing usages. I left one comment on the code.

File: shell/src/main/java/tachyon/shell/TfsShell.java [hunk 2]
```diff
@@ -97,33 +99,10 @@ private void loadCommands() {
...
-    System.out.println("       [rm <path>]");
-    System.out.println("       [rmr <path>]");
-    System.out.println("       [setTTL <path> <time to live(in milliseconds)>]");
-    System.out.println("       [unsetTTL <path>]");
-    System.out.println("       [tail <path>]");
-    System.out.println("       [touch <path>]");
-    System.out.println("       [unmount <tachyonPath>]");
-    System.out.println("       [unpin <path>]");
+    SortedSet<String> sortedCmds = new TreeSet<String>(mCommands.keySet());
+    for (String cmd: sortedCmds) {
```
Comment (calvinjia) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: `cmd : sortedCmds`

**Comment by leohjwang** [comment]
Just updated per @aaudiber and @calvinjia 's comments

**Comment by aaudiber** [comment]
@leohjwang It looks like you've merged master into your branch but haven't addressed the comments yet - is there another commit incoming?

**Comment by leohjwang** [comment]
@aaudiber sorry my bad. just had kicked another commit.

**Comment by aaudiber** [comment]
LGTM

**Comment by uronce-cc** [comment]
LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**